components/brave_ads/common/features.cc

```diff
@@ -63,7 +63,7 @@ constexpr char kFieldTrialParameterNotificationAdDarkModeBackgroundColor[] =
 constexpr char kDefaultNotificationAdDarkModeBackgroundColor[] = "202327";
 
 constexpr char kFieldTrialParameterShouldAttachNotificationAdToBrowserWindow[] =
-    "should_attached_ad_notification_to_browser_window";
+    "should_attach_ad_notification_to_browser_window";
```

Comment:
Maybe we can choose a proper naming here. See https://github.com/brave/brave-browser/issues/22216#issuecomment-1509323628

Reply:
Agreed to cleanup common features in a follow up PR. Thanks